### ISSUE TITLE
Support for Mobile assignment view 

### ISSUE THREAD

**Issue by pushyamig** [tl-its-umich-edu/student-dashboard-django#124]

**Comment by jonespm** [comment]
This sound like it might be something for after the Pilot?

**Comment by pushyamig** [comment]
Yes I think this needs  little bit of reading and doing it right way supporting both desktop and mobile view and hacking it

**Comment by jennlove-um** [comment]
UI/UX mobile expertise is needed here to translate the views to one that is better for mobile.
